pokemongo_bot/__init__.py

```diff
@@ -760,8 +760,18 @@ def _setup_logging(self):
         logging.getLogger("pgoapi").setLevel(log_level)
         logging.getLogger("rpc_api").setLevel(log_level)
 
-        if self.config.logging_clean and not self.config.debug:
-            formatter = Formatter(fmt='[%(asctime)s] %(message)s', datefmt='%H:%M:%S')
+        if self.config.logging:
+            logging_format = '%(message)s'
+            logging_format_options = ''
+            if ('show_log_level' in self.config.logging and self.config.logging['show_log_level']) or 'show_log_level' not in self.config.logging:
```

Comment:
That's what it does. First check is for "is in config and set true", second check is OR "not in config"